.gitignore

```diff
@@ -128,5 +128,5 @@ dmypy.json
 # Pyre type checker
 .pyre/
 
-# Visual studio
-.vscode/
```

Comment:
Is this not needed?

Reply:
We can put recommended extensions and settings in this folder in the future.

Reply:
Sounds good, so everybody will use same extensions.